docs/workflows/transitions/postprocess/postprocess/utils.py

```diff
@@ -35,3 +35,16 @@ def convert_predictions_to_v2(predictions):
             result[label] = sorted(result[label], key=lambda p: -p['confidence'])
 
     return result
+
+
+def to_validated_format(predictions):
```

Comment:
If this function is only intended to be used for v2 predictions I suggest we name the input argument thereafter. 
```suggestion
def to_validated_format(predictions_v2):
```